nservicebus/operations/performance-counters_counters_core_[6].partial.md

```diff
@@ -0,0 +1,7 @@
+include: performance-counters-criticaltime
+
+include: performance-counters-note
+
+include: performance-counters-slaviolation
```

Comment:
> include: performance-counters-slaviolation

wouldn't it be simpler to just add it as text in the main page? the link is not detected in the includes, and if we don't mind changing the order here then that would be the simplest solution